### DISCUSSION TITLE
Curating the editor experience deep dive posts

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#16] [Topic Ideas]
Similar to the pathways for adopting FSE features, it would be great to have content on what curating the editor experience looks like: https://developer.wordpress.org/block-editor/how-to-guides/curating-the-editor-experience/  I could see this being split between theme.json curation, locking API items, and removing/controlling access.

**Comment by bph** [comment]
Thank you @annezazu - Great ideas!

**Comment by bph** [comment]
I convert to a discussion to get more input and have questions answered, until we have more concrete outline a post or posts.
